node/libs/crypto/src/bls12_381/mod.rs

```diff
@@ -177,15 +200,15 @@ impl AggregateSignature {
     pub fn verify<'a>(
         &self,
         msgs_and_pks: impl Iterator<Item = (&'a [u8], &'a PublicKey)>,
```

Comment:
now the messages that are passed can be large and therefore indexing data structures like BTreeMap with messages will be inefficient.

Reply:
or maybe it is not that bad, nvm